Source/Scene/Model.js

```diff
@@ -1453,6 +1453,103 @@ define([
 
         return getStringFromTypedArray(loadResources.getBuffer(bufferView));
     }
+    
+    function modifyShaderForExtensions(shader, programName, model) {
```

Comment:
It's possible that one shader could be used for two different primitives, one with compressed attributes and one without compressed attributes.  Is this taken into account?  Add a unit test.